testing/peervpn/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Markus Juenemann <markus@juenemann.net>
+# Maintainer: Markus Juenemann <markus@juenemann.net>
+pkgname=peervpn
+pkgver=0.044
+pkgrel=0
+pkgdesc="PeerVPN builds virtual ethernet networks between multiple computers"
+url="https://peervpn.net"
+arch="all"
+license="GPL3"
+depends="openssl zlib"
+makedepends="$depends openssl-dev zlib-dev linux-headers"
+install=""
+subpackages=""
+source="
+        $url/files/$pkgname-0-044.tar.gz
+	peervpn.initd
+	"
+
+# peervpn has a strange versioning scheme.
+builddir="$srcdir/$pkgname-0-044"
+
+build() {
+
+	# Must copy these into fakeroot?
+	install -m 755 $pkgname.initd "$builddir"/$pkgname.initd
+
+	cd "$builddir"
+	make
+}
+
+package() {
+ 	cd "$builddir"
+
+	install -Dm755 $pkgname "$pkgdir"/usr/sbin/$pkgname \
+		|| return 1
+
+	install -Dm600 $pkgname.conf "$pkgdir"/etc/$pkgname.conf \
+		|| return 1
+
+	install -Dm755 $pkgname.initd "$pkgdir"/etc/init.d/$pkgname \
```

Comment:
Use $srcdir for the source file so it get's picked out without the extra install step in build().